docs/migrating/migrating.md

```diff
@@ -36,7 +36,7 @@ FFmpeg or Avconv is required for media transcoding. You can get it from [https:/
 $ npm install ffmpeg-static
 ```
 
-You can get avconv from [https://libav.org/download](https://libav.org/download).
+You can get avconv from [npmjs.org/package/avconv](https://npmjs.org/package/avconv).
```

Comment:
avconv from npm wont work as prism-media does not support it. The official website for avconv was libav.org but looks like it is no longer available. We should mention the following instead:

```suggestion
If you want to use avconv instead of ffmpeg, install it on your system or place avconv executable at the root of your project.
```